src/ParameterTable.ts

```diff
@@ -199,6 +204,10 @@ export class ParameterTable {
     // TODO: move to Eagle.ts?
     //       doesn't seem to depend on any ParameterTable state, only Eagle state
     static getNodeLockedState = (field:Field) : boolean => {
+        if (Setting.find(Setting.BOTTOM_WINDOW_MODE).value() === Eagle.BottomWindowMode.GraphConfigAttributesTable){
```

Comment:
i think this is fine. maybe add a comment that this is for the graph config table, the fields of which are always editable. 
We might have to revisit whether that is the case when we revamp ui-modes/permissions in the future

Reply:
I've added a comment to the code:

`// this handles a special case where EAGLE is displaying the "Graph Configuration Attributes Table"
  // all the fields shown in that table should be locked (readonly)`

This code is meant to make the "Attribute Name" field in the Graph Configuration Attributes Table" readonly. Since it doesn't really make sense to be able to rename a field when looking at a graph config.